.env.io-developer-portal.development

```diff
@@ -3,3 +3,4 @@ IO_DEVELOPER_PORTAL_BACKEND=https://developerportal-backend.io.italia.it
 IO_DEVELOPER_PORTAL_APIM_BASE_URL=https://io-dev-apim-01.azure-api.net/api/v1
 IO_DEVELOPER_PORTAL_BASE_URL=/
 IO_DEVELOPER_PORTAL_PUBLIC_PATH=/
+IO_LOGO_PATH=https://assets.cdn.io.italia.it/logos
```

Comment:
This is a PROD path so if we have to replicate this in a test environment we have to modify it.